test/images/sample-device-plugin/sampledeviceplugin.go

```diff
@@ -167,6 +187,16 @@ func main() {
 		if err := dp1.Stop(); err != nil {
 			panic(err)
 		}
+
+		// Remove CDI file
+		if cdi != "" {
```

Comment:
Does removing the file in a `defer` statement instead make sense?

Reply:
thank you for the suggestion! done.